source/docs/articles/sites/logs.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Log Files on Pantheon  
+description: Use log files to identify errors, track response times, analyze visitors and more on your WordPress or Drupal site.
+keywords: log, access log, nginx access log, nginx log, nginx access, nginx error, nginx error log, php error, php error log, php fpm error, php fpm, php error, php slow, php slow log, pyinofity log, pyinofity, watcher log, watcher
+draft: true
+---
+Log files track and record your site's activity which help you find, debug, and isolate current or potential problems on your site. These logs can also be used to compile statistics on your visitors and referral traffic.
+
+Each environment (Multidev, Dev, Test, and Live) has their own respective log files, which can be obtained via SFTP.
+
+## Download Raw Webserver Log Files
+1. Access the site Dashboard and desired environment (Mulidev, Dev, Test, or Live).
+2. Click **Connection Info** and copy the **SFTP Command Line** command.
+3. Open a terminal window and paste the SFTP connection command.
+4. Run the following SFTP command in terminal:
+
+   ```nohighlight
+   get -r logs
+   ```
+
+You now have a local copy of the logs directory, which contains the following:
+```nohighlight
+├── logs
+    └──nginx-access.log
+    └──nginx-error.log
+    └──php-error.log
+    └──php-fpm-error.log
+    └──php-slow.log
+    └──pyinotify.log
+    └──watcher.err
+    └──watcher.log
+```
+You can automate this process with the following script:
```

Comment:
Can we remove lines 35-45 and just link to the other doc? Or keep it here and not have the separate doc.